src/client/services/server-api/index.js

```diff
@@ -27,23 +27,15 @@ const ServerAPI = {
   },
 
   getGoInformation(goID) {
-    return fetch(`https://www.ebi.ac.uk/QuickGO/services/ontology/go/search?query=GO%3A${goID.replace("GO:", "")}&limit=1&page=1`, {method: 'GET', timeout: 100})
+    return fetch(`https://www.ebi.ac.uk/QuickGO/services/ontology/go/search?query=GO%3A${goID}&limit=1&page=1`, {method: 'GET', timeout: 100})
```

Comment:
Rethrow any errors for the UI to deal with. See https://github.com/PathwayCommons/app-ui/pull/962#issuecomment-411498990

Reply:
It's simpler to remove the catch altogether rather than rethrow within the catch.

Reply:
Remove `catch`